purview/models/koji.py

```diff
@@ -0,0 +1,55 @@
+from neomodel import (
+    StructuredNode, StringProperty, IntegerProperty, UniqueIdProperty,
+    BooleanProperty, DateTimeProperty, FloatProperty, RelationshipTo)
+
+
+class KojiBuild(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')
+    tasks = RelationshipTo('KojiTask', 'TRIGGERS')
+    creation_event_id = IntegerProperty(required=True)
+    volume_id = IntegerProperty()
+    owner_id = IntegerProperty()
+    package_id = IntegerProperty(required=True)
+    task_id = IntegerProperty(required=True)
+    build_id = IntegerProperty()
+    name = StringProperty()
```

Comment:
Check what this actually is.  :p

Maybe remove.

Reply:
Actually, this seems to be related to "version" and "release" below.  We probably need to keep it to construct the unique NVR of the build (this is how people talk about builds, as NVRs).

Reply:
`name` and `package_name` are the same thing. If Package is being moved into its own node, then these can both be removed. Unless there's value to duplicating the name in the KojiBuild (for query/filter reasons)?